trax/data/tf_inputs_test.py

```diff
@@ -591,21 +591,50 @@ def test_create_bert_inputs(self):
     create_inputs_2 = tf_inputs.CreateBertInputs(True)
     for res in create_inputs_1(zip(inputs_sentences_1, labels)):
       values, segment_embs, _, label, weight = res
-      self.assertTrue((values == np.array([101, 100, 150, 200, 102])).all())
-      self.assertTrue((segment_embs == np.zeros(5)).all())
+      self.assertAllEqual(values, np.array([101, 100, 150, 200, 102]))
+      self.assertAllEqual(segment_embs, np.zeros(5))
       self.assertEqual(label, np.int64(1))
       self.assertEqual(weight, np.int64(1))
 
     for res in create_inputs_2(
         zip(inputs_sentences_1, inputs_sentences_2, labels)):
       values, segment_embs, _, label, weight = res
-      self.assertTrue(
-          (values == np.array([101, 100, 150, 200, 102, 300, 500, 102])).all())
+      self.assertAllEqual(values, np.array([101, 100, 150, 200, 102, 300, 500, 102]))
       exp_segment = np.concatenate((np.zeros(5), np.ones(3)))
-      self.assertTrue((segment_embs == exp_segment).all())
+      self.assertAllEqual(segment_embs, exp_segment)
       self.assertEqual(label, np.int64(1))
       self.assertEqual(weight, np.int64(1))
 
+  def test_mask_random_tokens(self):
+    test_case_row = np.array([101]*100 + [1001]*100)
+    test_case = [(test_case_row.copy(), )]
+
+    out, original_tokens, token_weights = next(tf_inputs.mask_random_tokens(test_case))
+    # test whether original tokens are unchanged
+    self.assertAllEqual(test_case_row, original_tokens)
+
+    self.assertEqual(1, token_weights.sum())
+    self.assertEqual(15, (token_weights > 0).sum())
+
+    # 101 is a special token, so only 1001 should be masked
+    self.assertAllEqual(out[:100], test_case_row[:100])
+
+    # no more than 15 tokens with mask
+    self.assertLessEqual((out == 103).sum(), 15)
+
+  def test_BertNextSentencePredictionInputs(self):
```

Comment:
let's name as `test_bert_next_sen .... inputs()`

Also how does the test work? I don't see wiki40b's input files in the testdata ...